cisco_secure_endpoint/assets/logs/cisco-secure-endpoint_tests.yaml

```diff
@@ -0,0 +1,262 @@
+id: cisco-secure-endpoint
+tests:
+  - sample: |-
+      {
+        "item" : "abc@xyz.com",
+        "audit_log_type" : "User",
+        "created_at" : "2024-06-19T10:16:22Z",
+        "audit_log_user" : "abc@xyz.com",
+        "event" : "sso_login",
+        "message" : "SCSO",
+        "audit_log_id" : "10bfd8f2-8569-4b78-8996-19acc900f4fc"
+      }
+    result:
+      custom:
+        audit_log_id: "10bfd8f2-8569-4b78-8996-19acc900f4fc"
+        audit_log_type: "User"
+        audit_log_user: "abc@xyz.com"
+        created_at: "2024-06-19T10:16:22Z"
+        event: "sso_login"
+        item: "abc@xyz.com"
+      message: "SCSO"
+      tags:
+      - "source:LOGS_SOURCE"
+
+  - sample: |-
+      {
+        "item" : "f_00020c",
+        "new_attributes" : {
+          "analysis_state" : "threat_analyses.threat_analysis.failed.description"
+        },
+        "audit_log_type" : "FetchedFile",
+        "created_at" : "2024-06-19T06:38:50Z",
+        "event" : "update"
+      }
+    result:
+      custom:
+        audit_log_type: "FetchedFile"
+        created_at: "2024-06-19T06:38:50Z"
+        event: "update"
+        item: "f_00020c"
+        new_attributes:
+          analysis_state: "threat_analyses.threat_analysis.failed.description"
+      message: |-
+        {
+          "item" : "f_00020c",
+          "new_attributes" : {
+            "analysis_state" : "threat_analyses.threat_analysis.failed.description"
+          },
+          "audit_log_type" : "FetchedFile",
+          "created_at" : "2024-06-19T06:38:50Z",
+          "event" : "update"
+        }
+      tags:
+      - "source:LOGS_SOURCE"
+  - sample: |-
+      {
+        "item" : "Unconfirmed 61021.crdownload",
+        "new_attributes" : {
+          "original_filename" : "f_000253",
+          "state" : "processing"
+        },
+        "audit_log_type" : "FetchedFile",
+        "created_at" : "2024-06-19T06:28:17Z",
+        "event" : "update",
+        "old_attributes" : {
+          "state" : "requested"
+        }
+      }
+    result:
+      custom:
+        audit_log_type: "FetchedFile"
+        created_at: "2024-06-19T06:28:17Z"
+        event: "update"
+        item: "Unconfirmed 61021.crdownload"
+        new_attributes:
+          original_filename: "f_000253"
+          state: "processing"
+        old_attributes:
+          state: "requested"
+      message: |-
+        {
+          "item" : "Unconfirmed 61021.crdownload",
+          "new_attributes" : {
+            "original_filename" : "f_000253",
+            "state" : "processing"
+          },
+          "audit_log_type" : "FetchedFile",
+          "created_at" : "2024-06-19T06:28:17Z",
+          "event" : "update",
+          "old_attributes" : {
+            "state" : "requested"
+          }
+        }
+      tags:
+      - "source:LOGS_SOURCE"
+  - sample: |2-
```

Comment:
There is a wrong character "2" here

Reply:
Thankyou for the comment. Have made the changes.